doc/rose-rug-task-run.html

```diff
@@ -362,41 +362,80 @@ <h2 id="rose-task-run.built-in-app.fcm_make">Built-in Application:
-    <p>The <code>fcm make</code> application will also look for a matching
-    <code>fcm_make2*</code> task in the suite which would normally be a
-    continuation of the <code>fcm make</code> command as a separate
-    <var>USER@HOST</var>. If so, will export an environment variable
-    <var>ROSE_TASK_MIRROR_TARGET</var> to the <code>fcm make</code> command to
-    substitute the mirror target.</p>
-
     <p>You can configure these applications with environment variables or
     settings in <var>rose-app.conf</var>. (Settings in <var>rose-app.conf</var>
     override their equivalent environment variables.)</p>
 
-    <p><kbd>use-pwd=true</kbd>: By default, the application changes the working
-    directory to <var>$ROSE_SUITE_DIR/share/$ROSE_TASK_NAME</var>. This option
-    will stop this, and the working directory is the normal working directory
-    of the task.</p>
+    <dl>
+      <dt><kbd>args=ARG ...</kbd> or the environment variable
+      <var>ROSE_TASK_OPTIONS</var></dt>
+
+      <dd>This can be used to pass extra arguments to the <code>fcm make</code>
+      command.</dd>
+
+      <dt><kbd>opt.jobs=N</kbd> or the environment variable
+      <var>ROSE_TASK_N_JOBS</var></dt>
+
+      <dd>This can be used to control the number of processes <code>fcm
+      make</code> would use in parallel. (default=4)</dd>
 
-    <p><kbd>opt.jobs=N</kbd> or the environment variable
-    <var>ROSE_TASK_N_JOBS</var>: This can be used to control the number of
-    processes <code>fcm make</code> would use in parallel. (default=4)</p>
+      <dt><kbd>make-name-cont=NAME</kbd></dt>
 
-    <p><kbd>args</kbd> or the environment variable
-    <var>ROSE_TASK_OPTIONS</var>: This can be used to pass extra arguments to
-    the <code>fcm make</code> command.</p>
+      <dd>Specify the context name of the continuation make. If the default
+      <code>fcm_make</code> &rarr; <code>fcm_make2</code> mapping is used, the
+      context name of the continuation make will be set to <code>2</code>. You
+      can specify an alternate context name if this is undesirable. The
+      continuation command will be invoked with the <code>--name=NAME</code>
+      option of <code>fcm make</code>.</dd>
+
+      <dt><kbd>mirror-step=STEP-NAME</kbd></dt>
+
+      <dd>Specify the name of the mirror step, if not <samp>mirror</samp>. The
+      application will normally look for a matching task in the suite (e.g.
+      <code>fcm_make</code> &rarr; <code>fcm_make2</code>) which will continue
+      the <code>fcm make</code> command at a remote <var>HOST</var>. If such a
+      task is found, it will add the configuration
+      <var>mirror.target=HOST:cylc-run/$ROSE_SUITE_NAME/share/$ROSE_TASK_NAME</var>
+      as an argument to the <code>fcm make</code> command to substitute the
+      mirror target. To switch off this feature, set <var>STEP-NAME</var> to a
+      null string, i.e. <kbd>mirror-step=</kbd>.</dd>
+
+      <dt><kbd>task-name-orig2cont=ORIG-NAME:CONT-NAME</kbd></dt>
```

Comment:
`names-orig-cont`, comma separated? It's not strictly a task name after all (often just part of a task name)